source/interprocedural_analyses/taint/modelParser.ml

```diff
@@ -3264,6 +3270,131 @@ let rec parse_statement
             unexpected_models;
           };
       ]
+  | {
+   Node.value =
+     If
+       {
+         If.body;
+         If.test =
+           {
+             Node.value =
+               ComparisonOperator
+                 {
+                   ComparisonOperator.right = { Node.value = Tuple tuple; _ };
+                   ComparisonOperator.operator;
+                   _;
+                 };
+             _;
+           };
+         If.orelse;
+       };
+   location;
+  } ->
+      let version_list =
+        List.map
+          ~f:(fun expr ->
+            match expr with
+            | { Node.value = Constant const; _ } -> (
+                match const with
+                | Integer integer -> integer
+                | _ -> 0)
+            | _ -> 0)
+          tuple
+      in
+      let major, minor, micro =
+        match version_list with
+        | [first; second; third; _] -> first, second, third
+        | [first; second; third] -> first, second, third
+        | [first; second] -> first, second, 0
+        | [first] -> first, 0, 0
+        | _ -> 0, 0, 0
```

Comment:
Can we create a function that parses a list of version numbers into a `PythonVersion.t`. Additionally, module `PythonVersion` can have a default version value of
```
{major_version = 0; minor_version = 0; micro_version = 0}
```

Reply:
I tried to do so :)